sdk/servicebus/azure-servicebus/api_review.py

```diff
@@ -0,0 +1,588 @@
+    ) -> None:
+    @classmethod
+    def from_queue(
+        cls,
+        fully_qualified_namepsace : str,
+        queue_name : str,
+        credential : TokenCredential,
+        **kwargs
+    ) -> SeviceBusSenderClient:
+    @classmethod
+    def from_topic(
+        cls,
+        fully_qualified_namespace : str,
+        topic_name : str,
+        credential : TokenCredential,
+        **kwargs
+    ) -> SeviceBusSenderClient:
+    @classmethod
+    def from_connection_string(
+        cls,
+        conn_str : str,
+        entity_name : str = None,
+        **kwargs
+    ) -> SeviceBusSenderClient:
+
+    def __enter__(self):
+    def __exit__(self):
+
+    def close(self) -> None:
+
+    def get_properties(self) -> Dict[str, Any]:
+
+    def list_session_ids(self) -> List[str]:
+
+    def send(
+        self,
+        messages,
+        session : str = None,
+        message_timeout : float = None
+    ) -> None:
+    def schedule(
+        self,
+        messages,
+        schedule_time : datetime,
+        session : str = None
+    ) -> List[int]:
+    def cancel_scheduled_messages(self, sequence_numbers : List[int]) -> None:
+
+
+class ServiceBusReceiverClient:
+    def __init__(
+        self,
+        fully_qualified_namespace : str,
+        entity_name : str,
+        credential: TokenCredential,
+        subscription_name : str = None,
+        session : str = None,
+        mode : ReceiveSettleMode = ReceiveSettleMode.PeekLock
+        logging_enable: bool = False,
+        http_proxy: dict = None,
+        transport_type: TransportType = None,
+        retry_total : int = 3,
+    ) -> None:
+    @classmethod
+    def from_queue(
+        cls,
+        fully_qualified_namespace : str,
+        queue_name : str,
+        credential : TokenCredential,
+        session: str = None,
+        mode : ReceiveSettleMode = ReceiveSettleMode.PeekLock
+        **kwargs
+    ) -> ServiceBusReceiverClient:
+    @classmethod
+    def from_topic_subscription(
+        cls,
+        fully_qualified_namespace : str,
+        topic_name : str,
+        subscription_name : str,
+        credential: TokenCredential,
+        session: str = None,
+        mode : ReceiveSettleMode = ReceiveSettleMode.PeekLock
+        **kwargs
+    ) -> ServiceBusReceiverClient:
+    @classmethod
+    def from_connection_string(
+        cls,
+        conn_str : str,
+        entity_name : str = None,
+        subscription_name : str = None,
+        session: str = None,
+        mode : ReceiveSettleMode = ReceiveSettleMode.PeekLock
+        **kwargs
+    )-> ServiceBusReceiverClient:
+
+    def __enter__(self):
+    def __exit__(self):
+
+    def close(self) -> None:
+
+    def get_properties(self) -> Dict[str, any]:
+
+    def __iter__(self):
+    def __next__(self):
+    def next(self):
+
+    def peek(
+        self,
+        count : int = 1,
+        start_from : int = None
+    ) -> List[PeekMessage]:
+    def receive_deferred_messages(
```

Comment:
I wonder if we should have consistent suffixes here:
`receive_messages` and `receive_deferred_messages`

Thoughts?

Reply:
`receive_messages` does look more consistent -- I'm in 😀

Reply:
as per discussion, keep `receive` for now.